.circleci/config.yml

```diff
@@ -25,17 +29,27 @@ jobs:
             - common-dependencies-{{ checksum "common/package.json" }}
             - common-dependencies-
       # install the new dependencies for all the packages
+      - run:
```

Comment:
It could make the build slightly faster, but it's actually testing two completely different things. The function have `admin` access to the DB so they don't really care about permissions.

Reply:
It is also true that rules are deployed as part of the functions deploy, so the two are somewhat related. But again, I'm not gonna make a case for it.

Reply:
That was a lie, rules are deployed as part of the client deploy... anyways, the fact that we maintain two versions of firebase-tools (and so the firestore emulator) is not great but we can live with it.